src/rdbms-connect/azext_rdbms_connect/_help.py

```diff
@@ -24,8 +22,26 @@
 example:
   - name: Connect to a flexible server using administrator username and password. Default database is used if not specified.
     text: az postgres flexible-server connect -n testServer -u username -p password -d postgres
-  - name: Connect to a flexible server and execute a query with results outputted in a table.
-    text: az postgres flexible-server connect -n testServer -u username -p password --querytext "select * from pg_user;" --output table
   - name: Connect to a flexible server and run queries interactively.
     text: az postgres flexible-server connect -n testServer -u username --interactive
 """
+
+helps['mysql flexible-server execute'] = """
+type: command
+short-summary: Connect to a flexible server.
+example:
+  - name: Connect to a flexible server and execute a query with results outputted in a table.
+    text: az mysql flexible-server connect -n testServer -u username -p password --querytext "select host, user from mysql.user;" --output table
+  - name: Connect to a flexible server and execute a sql file.
+    text: az mysql flexible-server connect -n testServer -u username -p password --file-path path_to_sql_file
+"""
+
+helps['postgres flexible-server execute'] = """
+type: command
+short-summary: Connect to a flexible server.
+example:
+  - name: Connect to a flexible server and execute a query with results outputted in a table.
+    text: az postgres flexible-server connect -n testServer -u username -p password --querytext "select * from pg_user;" --output table
+  - name: Connect to a flexible server and execute a sql file.
+    text: az mysql flexible-server connect -n testServer -u username -p password --file-path path_to_sql_file
```

Comment:
the commands should be `execute`?

Reply:
that was silly mistake. Thanks a lot for catching this!